src/data/Data.js

```diff
@@ -147,7 +147,12 @@ Data.prototype = {
 
 		// Remove the expando if there's no more data
 		if ( key === undefined || jQuery.isEmptyObject( cache ) ) {
-			delete owner[ this.expando ];
+			// Support: Chrome <=35 - 45
+			// Webkit & Blink performance suffers when deleting properties
+			// from DOM nodes, so set to undefined instead
+			// https://code.google.com/p/chromium/issues/detail?id=378607
+			// https://github.com/jquery/jquery/pull/1664
```

Comment:
No need to reference the other PR; followup will start with `blame` and/or `log` anyway.